lib/bdev/bdev.c

```diff
@@ -857,6 +859,36 @@ spdk_bdev_reset(struct spdk_bdev *bdev, enum spdk_bdev_reset_type reset_type,
 	return rc;
 }
 
+struct spdk_bdev_io *
```

Comment:
We'll add the channel to the interface, but reset still resets channel to NULL in __submit_request. Should we keep the same logic for passthru or take it out?